Deploy/templates/admin.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: cbs-admin-db
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cbs-admin-db
+        environment: dev
+    spec:
+      containers:
+      - name: cbs-admin-db
+        image: mongo
+        ports:
+        - containerPort: 27017
+          name: mongo
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cbs-admin-db
+spec:
+  ports:
+  - port: 27017
+  selector:
+    app: cbs-admin-db
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: cbs-admin-backend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cbs-admin-backend
+        environment: dev
+    spec:
+      containers:
+      - name: cbs-admin-backend
+        image: karolikl/cbs-admin-backend:{{ .Values.services.admin.backend.version }}
+        ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.services.admin.backend.service }}
+spec:
+  type: NodePort
+  ports:
+  - port: 80
+  selector:
+    app: cbs-admin-backend
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: cbs-admin-frontend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cbs-admin-frontend
+        environment: dev
+    spec:
+      containers:
+      - name: cbs-admin-frontend
+        image: karolikl/cbs-admin-frontend:{{ .Values.services.admin.frontend.version }}
```

Comment:
Is his in your container registry?

Reply:
Yes, we started moving it to another image repository during the last codeathon, but we didn't have the correct access rights. Will make sure we update asap :)